bigtable/tests/data_integration_test.cc

```diff
@@ -30,66 +92,231 @@ int main(int argc, char* argv[]) try {
   std::string const project_id = argv[1];
   std::string const instance_id = argv[2];
   std::string const table_name = argv[3];
```

Comment:
Since you now create a new table for each test the table id is no longer needed, right?  Please remove it as a required argument.

Reply:
Fixed!